go.mod

```diff
@@ -5,8 +5,10 @@ go 1.18
 require (
 	github.com/dustin/go-humanize v1.0.0
 	github.com/flynn/go-docopt v0.0.0-20140912013429-f6dd2ebbb31e
-	github.com/secure-systems-lab/go-securesystemslib v0.4.0
-	github.com/stretchr/testify v1.7.4
+	github.com/google/gofuzz v1.2.0
+	github.com/onsi/gomega v1.18.1 // indirect
+	github.com/secure-systems-lab/go-securesystemslib v0.3.1
+	github.com/stretchr/testify v1.7.1
```

Comment:
Why are these two dependencies moving back a version?

Reply:
I need to rebase to master again.